samples: bluetooth: HID SCI support in peripheral HID samples (#28937)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_hids_keyboard/README.rst

      LED 3:
         Indicates if an NFC field is present.

HID Shorter Connection Intervals (SCI)
--------------------------------------

If the sample is built with ``-DFILE_SUFFIX=hid_sci``, support for HID Shorter Connection Intervals (SCI) is enabled.
This feature allows the device to enter a specific HID SCI mode when requested by the central.

> The HID SCI modes are sets of connection parameters which can be used to optimize the connection for the specific use case.

Reviewer: ```suggestion
The HID SCI modes are sets of connection parameters that you can use to optimize the connection for a specific use case.
```

.. options-from-kconfig::
   :show-type:

.. peripheral_hids_hid_sci_start

HID Shorter Connection Intervals (SCI)
--------------------------------------

> If the sample is built with ``-DFILE_SUFFIX=hid_sci``, support for HID Shorter Connection Intervals (SCI) is enabled.

Reviewer: ```suggestion
If you build the sample with ``-DFILE_SUFFIX=hid_sci``, support for HID Shorter Connection Intervals (SCI) is enabled.
```

.. include:: /includes/ipc_radio_conf.txt

> To build the sample with HID SCI support, build with ``-DFILE_SUFFIX=hid_sci``

Reviewer: ```suggestion
To build the sample with HID SCI support, use ``-DFILE_SUFFIX=hid_sci``
```
